### PR TITLE
Fix: Program Page Feature Image code

### PR DESCRIPTION
**Pull request by Claurt07**
Changes: 
-  The image could not be changed through Siteleaf so I changed the code --Previously the image was on the CSS file.

### CODE REVIEW

**Comment by Claurt07** [comment]
@kshitijrajsharma Just a quick fix for the image section on the Program page :)

File: _sass/base.scss [hunk 1]
```diff
@@ -5894,6 +5918,178 @@ th, td {
 
 }
 
+.half-block-tools {
```
Comment (kshitijrajsharma) [inline _sass/base.scss:5921]: Intended css ?

**Review by kshitijrajsharma — APPROVED** [review]

File: _sass/base.scss (hunk 1 above)
Comment (Claurt07) [reply _sass/base.scss:5921]: Yes it is, Although I havent committed the related HTML yet

**Comment by kshitijrajsharma** [comment]
Its good practice to  include changes that are related to that fix only . There is one .md change and basecss which doesn't belong to this fix

If you want to include extra changes in PR then you can mention them in description. 

You can keep working on other things for same file , You just need to branchout with base from this PR branch

Its okay for now, tips for next changes 😉

**Pull request closed without merging**